specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.1/keys.json

```diff
@@ -988,9 +988,7 @@
             "EC",
             "EC-HSM",
             "RSA",
-            "RSA-HSM",
-            "oct",
```

Comment:
"oct" was actually in 7.0 so you can't remove it. That would be a breaking change. Presumably, the Key Vault service already responds with an error if you try to use it.